test/unittests/libp2p/net.cpp

```diff
@@ -61,8 +61,8 @@ struct TestNodeTable: public NodeTable
     static std::vector<std::pair<Public, uint16_t>> createTestNodes(unsigned _count)
     {
         std::vector<std::pair<Public, uint16_t>> ret;
-        asserts(_count < 1000);
-
+        asserts(_count <= 1000);
```

Comment:
Why is this capped at 1000? Is 1000 an arbitrary limit (but we just need some limit)?

Reply:
Yes, it's arbitrary, I  guess we could also just remove it